src/Listeners/GiveMoney.php

```diff
@@ -59,46 +63,79 @@ public function giveMoney(?User $user, float $money): bool
         return false;
     }
 
+    public function postGiveMoney(?User $user, float $money, Post $post)
+    {
+        if (!is_null($user)) {
+            $permissions = true;
+            if ($post) {
+                $discussionTags = $post->discussion->tags;
+                foreach ($discussionTags as $tag) {
+                    if ($user->hasPermission("tag{$tag->id}.discussion.money.disable_money") && !$user->isAdmin()) {
```

Comment:
```suggestion
                    if ($user->hasPermission("tag{$tag->id}.discussion.money.disable_money")) {
```

I would prefer to not exclude admins here. It can already be done in the Permissions page, and it is more flexible there.

Reply:
Because the Flarum framework automatically selects the administrator and does not allow unchecking the administrator option, I added this piece of code to enable the possibility of an exemption for the administrator. Otherwise, the administrator account would be unable to access money site-wide.

Reply:
Looks good to me, I didn't think that it was uncheckable.